deno_webgpu/adapter.rs

```diff
@@ -125,8 +125,13 @@ impl GPUAdapter {
             return Err(CreateDeviceError::RequiredFeaturesNotASubset);
         }
 
-        let required_limits =
-            serde_json::from_value(serde_json::to_value(descriptor.required_limits)?)?;
+        // When support for compatibility mode is added, this will need to look
+        // at whether the adapter is "compatibility-defaulting" or "core-defaulting",
+        // and choose the appropriate set of defaults.
```

Comment:
suggestion: Could we link this to an issue, so an inquisitive person can follow along with `compat` development?

Reply:
Filed and linked #8124.